openedx/core/djangoapps/coursegraph/signals.py

```diff
@@ -0,0 +1,15 @@
+"""
+Signal handlers for the CourseGraph application
+"""
+from django.dispatch.dispatcher import receiver
+from xmodule.modulestore.django import SignalHandler
+
+from openedx.core.djangoapps.coursegraph.utils import CourseLastPublishedCache
+
+
+@receiver(SignalHandler.course_published)
```

Comment:
I don't know this decorator. Will this have a memory leak? No idea..just asking.